jiant/scripts/download_data/runscript.py

```diff
@@ -6,9 +6,10 @@
 import jiant.scripts.download_data.datasets.xtreme as xtreme_download
 import jiant.scripts.download_data.datasets.files_tasks as files_tasks_download
 from jiant.tasks.constants import GLUE_TASKS, SUPERGLUE_TASKS, XTREME_TASKS, BENCHMARKS
+from jiant.scripts.download_data.constants import SQUAD_TASKS, INCOMPATIBLE_NLP_TASKS
 
-NLP_DOWNLOADER_TASKS = GLUE_TASKS | SUPERGLUE_TASKS
-SUPPORTED_TASKS = NLP_DOWNLOADER_TASKS | XTREME_TASKS | {"squad_v1", "squad_v2"}
+NLP_DOWNLOADER_TASKS = GLUE_TASKS | SUPERGLUE_TASKS - INCOMPATIBLE_NLP_TASKS
```

Comment:
Do we need to refer to these as "incompatible NLP tasks"? We should simply designate that a certain set of tasks are downloaded via NLP, and the others will be downloaded using other methods.

Reply:
What do you think about this change?
```suggestion
NLP_DOWNLOADER_TASKS = GLUE_TASKS | SUPERGLUE_TASKS - DIRECT_DOWNLOAD_TASKS
```